.github/scripts/check-for-gitjob-changes.sh

```diff
@@ -4,11 +4,11 @@ set -ue
 : "${MAIN_BRANCH:=master}"
 
 if [ "$(git diff --name-only "${MAIN_BRANCH}" -- "charts/fleet/charts/gitjob/" | wc -l)" -gt "0" ]; then
-    if ! $(git diff "${MAIN_BRANCH}" -- "charts/fleet/charts/gitjob/Chart.yaml" | grep -qE 'version: [.0-9]+'); then
-        echo "Gitjob needs to be updated in the 'rancher/gitjob' repo first and"
-        echo "then the new Gitjob release can be added to Fleet.\n"
+    if ! git diff "${MAIN_BRANCH}" -- "charts/fleet/charts/gitjob/Chart.yaml" | grep -qE 'version: [.0-9]+'; then
+        printf "Gitjob needs to be updated in the 'rancher/gitjob' repo first and"
+        printf "then the new Gitjob release can be added to Fleet.\n"
```

Comment:
No need to change the echo to printf and not even  use a format string. Just remove the `\n`. echo adds a newline by default anyway. Just add more `echo`s for more newlines..